cmd/kube-controller-manager/app/storageversionmigrator.go

```diff
@@ -79,6 +86,11 @@ func newSVMController(ctx context.Context, controllerContext ControllerContext,
 		return nil, err
 	}
 
+	factory := externalversions.NewSharedInformerFactory(crdClientset, 0)
```

Comment:
I don't like creating a shared informer just for SVM but I think this is better than listing CRDs each time we reconcile. Any suggestions here much appreciated :)